### DISCUSSION TITLE
mmWave Breathing and Heartbeat Module | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#54] [Q&A]
# mmWave Breathing and Heartbeat Module | Seeed Studio Wiki

mmWave Breathing and Heartbeat Module

https://wiki.seeedstudio.com/Radar_MR60BHA1/#firmware-version-updates

**Comment by vladkozlov69** [comment]
By the way, the cheapest JLink-OB works with this module very well. I was thinking that M4 processor requires at least full-featured JLink-9, but tried it just out of curiosity :)

**Reply by MatthewJeffson** [reply]
Haha, amazing!

**Comment by daejin84** [comment]
Is there any wat to get the real raw data??

**Reply by daejin84** [reply]
or any way to strengthen radar power??

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
